sqldb/migrations.go

```diff
@@ -381,6 +387,30 @@ func ApplyMigrations(ctx context.Context, db *BaseDB,
 			"(dirty=%v) as base version", currentVersion, dirty)
 	}
 
+	// Due to an a migration issue in v0.19.0-rc1 we may be at version 2 and
+	// have a dirty schema due to failing migration 3. If this is indeed the
+	// case, we need to reset the dirty flag to be able to apply the fixed
+	// migration.
+	// NOTE: this could be removed as soon as we drop v0.19.0-beta.
+	if version == 2 {
```

Comment:
I don't understand why we needed to roll a custom migration system here. This is exactly what we wanted to get away from by moving to `sqlc`. 

Now we have this piece mail code here that needs to manually alter migration versions vs just letting the tool entirely handle the up+down movement.